src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngine.cs

```diff
@@ -415,6 +416,14 @@ public string GetAbsolutePath(string executingFilePath, string pagePath)
                 var viewStartPages = isMainPage ?
                     GetViewStartPages(relativePath, expirationTokens) :
                     EmptyArray<ViewLocationCacheItem>.Instance;
+                if (factoryResult.IsPrecompiledView)
+                {
+                    _logger?.PrecompiledViewFound(relativePath);
```

Comment:
`_logger` isn't optional here, so you shouldn't need the null-check operator here.